views/login.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <script src="/static/admin.js"></script>
+    <link rel="stylesheet" href="/static/styles.css" />
+    <title>Login</title>
+  </head>
+  <body>
+    <style>
+      .error {
+        text-align: center;
+      }
+    </style>
+    <h1>
+      <center><strong>Login</strong></center>
+    </h1>
+    <div class="container">
+      <% // If there is an error in our view model, render it here %>
+      <div class="error"><%= error %></div>
```

Comment:
This error variable gets replaced by express's view engine before it's sent to the user.